pKa_challenge_instructions.md

```diff
@@ -130,6 +130,15 @@ Newly added microstates will also be shared with participants and microstates li
 Please do not create a microstate ID yourself. 
 It is important that challenge organizers assign unique microstate IDs and keep track.  
 
+### Updates on microstates lists
+
+Microstate SMILES strings and microstate IDs can be found in `physical_properties/pKa/microstates/` directory. 
```

Comment:
Whenever possible, I think it is useful to provide links so people don't have to click around to figure out where you're pointing them. 

If this directory existed before this PR, could you make this a hyperlink so people can click on the path?
If the directory is new a link will need to be added after this PR.

Reply:
I agree. I added a link.

Reply:
@bannanc FTR, one can add links in PRs that add the directory (a link is just formatted text). Relative links (like here) will be active even before merging since they'll just refer elsewhere in the branch; absolute links might not be active until merging.